src/python/CRABClient/Commands/status.py

```diff
@@ -355,7 +359,7 @@ def checkUserJobids(self, statusCacheInfo, userJobids):
         if wrongJobIds:
             raise ConfigurationException("The following jobids were not found in the task: %s" % wrongJobIds)
 
-    def printDetails(self, dictresult, jobids=None, quiet=False):
+    def printDetails(self, dictresult, jobids=None, quiet=False, maxMemory=2000, maxJobRuntime=1315):
```

Comment:
default valules should not be hardwired in two places. Need to pick maxMemory and maxJobRuntime via e.g.
`from CRABClient.ClientMapping import parametersMapping, getParamDefaultValue`